main.go

```diff
@@ -319,7 +337,7 @@ func convertFile(file *descriptor.FileDescriptorProto) ([]*plugin.CodeGeneratorR
 			return nil, err
 		}
 
-		jsonSchema, err := json.Marshal(schema)
+		jsonSchema, err := json.MarshalIndent(schema, "", " ")
```

Comment:
I had a similar issue as in #9: marshaling to compact JSON makes it a real pain to do code reviews on schema changes. Instead of adding another option, I chose to just make it always emit non-compact JSON.

Reply:
I figured this should be optional, because I don't know how many unit-tests out there may depend on the current output format.